src/DataFlowGraphModel.cpp

```diff
@@ -134,7 +134,7 @@ void DataFlowGraphModel::addConnection(ConnectionId const connectionId)
 {
     _connectivity.insert(connectionId);
 
-    Q_EMIT connectionCreated(connectionId);
```

Comment:
We can't simply stop emitting this signal.
It is attached to the `BasicGraphicsScene` which in turn repaints all the connected nodes.